### ISSUE TITLE
Original stack trace for 'bert/encoder/layer_2/attention/self/MatMul'： 什么问题

### ISSUE THREAD

**Issue by impltrait** [ProHiryu/bert-chinese-ner#43]
Original stack trace for 'bert/encoder/layer_2/attention/self/MatMul': File "BERT_NER.py", line 621, in <module> tf.app.run() File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\platform\app.py", line 40, in run _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\absl\app.py", line 300, in run _run_main(main, args) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\absl\app.py", line 251, in _run_main sys.exit(main(argv)) File "BERT_NER.py", line 554, in main estimator.train(input_fn=train_input_fn, max_steps=num_train_steps) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 2871, in train saving_listeners=saving_listeners) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 367, in train loss = self._train_model(input_fn, hooks, saving_listeners) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1158, in _train_model return self._train_model_default(input_fn, hooks, saving_listeners) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1188, in _train_model_default features, labels, ModeKeys.TRAIN, self.config) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 2709, in _call_model_fn config) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1146, in _call_model_fn model_fn_results = self._model_fn(features=features, **kwargs) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 2967, in _model_fn features, labels, is_export_mode=is_export_mode) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 1549, in call_without_tpu return self._call_model_fn(features, labels, is_export_mode=is_export_mode) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 1867, in _call_model_fn estimator_spec = self._model_fn(features=features, **kwargs) File "BERT_NER.py", line 411, in model_fn num_labels, use_one_hot_embeddings) File "BERT_NER.py", line 361, in create_model use_one_hot_embeddings=use_one_hot_embeddings File "C:\Users\leade\Desktop\bert-chinese-ner-v2\bert\modeling.py", line 216, in __init__ do_return_all_layers=True) File "C:\Users\leade\Desktop\bert-chinese-ner-v2\bert\modeling.py", line 844, in transformer_model to_seq_length=seq_length) File "C:\Users\leade\Desktop\bert-chinese-ner-v2\bert\modeling.py", line 701, in attention_layer attention_scores = tf.matmul(query_layer, key_layer, transpose_b=True) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\util\dispatch.py", line 180, in wrapper return target(*args, **kwargs) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\ops\math_ops.py", line 2609, in matmul return batch_mat_mul_fn(a, b, adj_x=adjoint_a, adj_y=adjoint_b, name=name) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\ops\gen_math_ops.py", line 1806, in batch_mat_mul_v2 "BatchMatMulV2", x=x, y=y, adj_x=adj_x, adj_y=adj_y, name=name) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 788, in _apply_op_helper op_def=op_def) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\util\deprecation.py", line 507, in new_func return func(*args, **kwargs) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\framework\ops.py", line 3616, in create_op op_def=op_def) File "D:\ProgramFiles\Anaconda3\envs\roots\lib\site-packages\tensorflow\python\framework\ops.py", line 2005, in __init__ self._traceback = tf_stack.extract_stack()

**Comment by yanqiangmiffy** [comment]
这个原因是什么呢？
